sfdoc/publish/models.py

```diff
@@ -287,3 +287,20 @@ def get_or_create_by_docset_id(cls, docset_id):
     @property
     def display_name(self):
         return self.name or self.docset_id
+
+
+class WhitelistedLinkset(models.Model):
+    name = models.CharField(max_length=100, unique=True, null=True)
+    urls = models.TextField()
+
+    @classmethod
+    def all_urls(cls):
+        all_urls = []
+        for obj in cls.objects.all():
+            for u in obj.urls.split("\n"):
+                all_urls.append(u.strip())
```

Comment:
How about `all_urls += [u.strip() for u in obj.urls.splitlines()]`

Reply:
Mildly prefer the way it is. I did consider a top-to-bottom list comprehension but thought that might be harder to read.

Also: list += bothers me because I know its creating a temporary list in addition to the one created by the list comprehension.

I could compromise on:

all_urls.extend(u.strip() for u in obj.urls.splitlines())

If you prefer that.

Reply:
It's fine as is.